nncf/quantization/fake_quantize.py

```diff
@@ -307,6 +307,9 @@ def calculate_scale_zero_point(
     """
     levels = level_high - level_low if narrow_range else level_high - level_low + 1
     scale = np.array((input_high - input_low) / (levels - 1)).astype(np.float32)
+    eps = np.finfo(scale.dtype).eps
+    # NOTE: adding machine epsilon to avoid division by zero
+    scale[np.abs(scale) < eps] = eps
```

Comment:
Have you assessed the impact of adding eps on ONNX PTQ?

Reply:
Pre-commit scope includes test for this functionality. https://github.com/openvinotoolkit/nncf/blob/eaddf18ee1d44457c65d76d02e16e90a25f9f828/tests/onnx/quantization/test_calculation_quantizer_params.py#L35
I assume it should be enough. @kshpv, please confirm

Reply:
@kshpv, added test for zero scale. In terms of ONNX format, everything looks fine, since scale and zero point are float32.
@alexsu52 the worst thing that can happen here is when all values in the channel are equal to a big float number. After division this by eps overflow may happen. But runtime should clip it to maximum float32. 
![image](https://github.com/openvinotoolkit/nncf/assets/4014476/69af3d87-a8a3-4932-bcf7-b0c0fac67b53)